tools/eslint-plugin-minecraft-linting/src/Rules/AvoidUnnecessaryCommand.test.ts

```diff
@@ -611,6 +611,21 @@ describe('AvoidUnnecessaryCommand', () => {
                         },
                     ],
                 },
+                {
+                    code: `const p = player.runCommandAsync('/inputpermission query @s movement enabled');`,
```

Comment:
Hmmm I'm not super familiar with lint rules (so feel free to correct me if I'm wrong or wait for Jake/Raph) but would this only work for that specific string? I think we'd want something a bit more general like just `/inputpermission` so if it appears anywhere we can suggest replace with script method

Reply:
It looks like the other ones don't do this though so I'm really not sure 😅